docs/how-to/index.rst

```diff
@@ -34,6 +34,16 @@ Learn how to manage `MLflow <Charmed MLflow_>`_ within your DSS environment.
 
    mlflow
 
+Leverage Intel GPUs
```

Comment:
Let's merge this into one section (Leverage GPUs), see my suggestion below. Then we add two bullet points, one for NVIDIA and another one for Intel